Userland/Libraries/LibGUI/Widget.cpp

```diff
@@ -891,16 +891,19 @@ Vector<Widget&> Widget::child_widgets() const
 void Widget::set_palette(const Palette& palette)
 {
     m_palette = palette.impl();
+    update();
```

Comment:
These changes are already in master: 5b79d53c893315f489c352b627071f5bd4c8dd36

Reply:
Removed commit